.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,71 @@
+name: Build PR-Preview
+
+on:
+  pull_request_review:
+    types: submitted
+
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: true
+
+env:
+  LAYOUT_EDITOR_BASE_URL: "/EmbeddedChat/pulls/pr-${{github.event.pull_request.number}}/layout_editor"
+  DOCS_BASEURL: "/EmbeddedChat/pulls/pr-${{github.event.pull_request.number}}/docs"
+
+jobs:
+  build:
+    if: github.event.review.state == 'approved' && (github.event.review.author_association == 'COLLABORATOR' || github.event.review.author_association == 'OWNER')
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
Remove outputs as discussed and tested (as it has no use)

Reply:
done